Academia-750/academia750-frontend-vue
# DBT-727 DBT-726 Materials

## Context


This PR contains new features for adding materials to workspaces but not uploading files yet


[Link](https://www.notion.so/tianlu/Frontend-Form-to-create-edit-a-file-eb9cb8eba6834dee9d2a24ac4c6073cd?pvs=4)
## Test
List of things to test

- Form to create edit a file
- Table to display the files of a workspace. Including filtering by workspace, category and tags
- 



- Create a workspace
- check list of workspace and count of materials
- click on materials to check uploaded materials



## Platform
Google chrome


## Proof of Concept


![Screenshot 2023-08-04 at 11 53 52](https://github.com/Academia-750/academia750-frontend-vue/assets/79788219/d6f339e9-309a-4709-ace8-85914a3ec3ee)
